Bottom: 474266e0219aa3d67b3054345c4b24d3e5e1ecdc
Top:    fab2fda3e5ef6d096661c2f345d7bc765b86efdf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 00:19:46 +0100

Refresh of misc-c-add-const-qualifier-to-

---

diff --git a/src/misc.c b/src/misc.c
index f6e01c5..ee7e26d 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -706,7 +706,7 @@ hole_counting_callback (const BoxType * b, void *cl)
  * within_area.
  */
 void
-CountHoles (int *plated, int *unplated, BoxType *within_area)
+CountHoles (int *plated, int *unplated, const BoxType *within_area)
 {
   HoleCountStruct hcs = {0, 0};
 
diff --git a/src/misc.h b/src/misc.h
index f625cbc..49246f1 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -59,7 +59,7 @@ bool IsLayerEmpty (LayerTypePtr);
 bool IsLayerNumEmpty (int);
 bool IsLayerGroupEmpty (int);
 bool IsPasteEmpty (int);
-void CountHoles (int *, int *, BoxType *);
+void CountHoles (int *, int *, const BoxType *);
 BoxTypePtr GetDataBoundingBox (DataTypePtr);
 void CenterDisplay (Coord, Coord);
 void SetFontInfo (FontTypePtr);
